src/mqt/bench/benchmark_generator.py

```diff
@@ -230,15 +252,14 @@ def start_benchmark_generation(self, create_circuit_function, parameters, file_p
 def get_benchmark(  # noqa: PLR0911, PLR0912, PLR0915
     benchmark_name: str,
     level: str | int,
-    circuit_size: int = None,
-    benchmark_instance_name: str = None,
+    circuit_size: int | None = None,
+    benchmark_instance_name: str | None = None,
     compiler: str | None = "qiskit",
-    compiler_settings: dict[str, dict[str, any]] | None = None,
+    compiler_settings: dict[str, dict[str, int | str]] | None = None,
     gate_set_name: str | None = "ibm",
     device_name: str | None = "ibm_washington",
-):
+) -> QuantumCircuit | Circuit:
```

Comment:
I don't think it would make it necessarily easier since this would mean to add another conversion to all `TKET` compilations. To be it feels more natural to return a `TKET::Circuit` when using the `TKET` compiler. But I also do not really mind changing this if you think returning the same format is advantageous.

Reply:
Agreed. Let's keep the TKET type.